assets/components/content-types/study-plan/study-plan-ie.scss

```diff
@@ -0,0 +1,42 @@
+@charset "utf-8";
+
+/* stylelint-disable */
+// This file is intended for IE 11 and has autoprefixer and stylelint off.
+
+// IE 11 table headings
+@include ie11('.table-like .cours-title') {-ms-grid-column: 2;}
```

Comment:
The result on IE11 - Windows 8.1 (browserstack):
![capture d ecran 2018-08-06 a 13 39 07](https://user-images.githubusercontent.com/4548432/43714942-200cb2e6-997f-11e8-83bf-ca8a08ed1a56.png)

Reply:
Ah shit... I tested on Windows 7, I think it looked better than this :(